### DISCUSSION TITLE
fetching issue with IOS

### DISCUSSION THREAD

**Discussion by vikramahir** [flutter-stripe/flutter_stripe#1295] [Q&A]
Uncategorized (Xcode): Command CompileSwift failed with a nonzero exit code

Swift Compiler Error (Xcode): 'data(for:delegate:)' is only available in iOS 15.0 or newer
/Users/dreamworld/MoistureStudio/Vikram/untitled/ios/Pods/StripePaymentsUI/StripePaymentsUI/StripePaymentsUI/Source/UI%20Componen
ts/PaymentMethodMessagingView.swift:296:55

**Comment by jonasbark** [comment]
What is the deployment target of your iOS project?

**Comment by vikramahir** [comment]
The deployment target of iOS is 13 and also I tried with 15. But still the same issue.
